pyaedt/modules/SolveSetup.py

```diff
@@ -562,7 +564,7 @@ def _setup(self, soltype, arg, newsetup=True):
                 self.omodule.EditAMIAnalysis(self.name,arg)
 
             else:
-                print("Not Implemented Yet")
+                print("Not implemented yet.")
```

Comment:
@maxcapodi78, this and the other `print("Not Implemented Yet")` above this should be:
```suggestion
                raise NotImplementedError("Solution type '{}' is not implemented yet".format(soltype))
```
Do you see any issue with this?